.eslintrc.json

```diff
@@ -514,6 +530,64 @@
       2,
       "never"
     ],
+    "padding-line-between-statements": [
+      2,
+      {
+        "blankLine": "always",
+        "prev": "directive",
+        "next": "*"
+      },
+      {
+        "blankLine": "always",
+        "prev": "import",
+        "next": "export"
+      },
+      {
+        "blankLine": "always",
+        "prev": [
+          "import",
+          "export"
+        ],
+        "next": [
+          "const",
+          "let",
+          "var",
+          "block-like"
+        ]
+      },
+      {
+        "blankLine": "always",
+        "prev": "*",
+        "next": "function"
+      },
+      {
+        "blankLine": "always",
+        "prev": "*",
+        "next": [
+          "block-like",
+          "multiline-block-like"
+        ]
+      },
+      {
+        "blankLine": "always",
```

Comment:
This and the following may not be necessary with block-like/multiline-block-like